integration_tests/cairo_zero_hints_tests/math_hints/signed_div_rem_test.cairo

```diff
@@ -0,0 +1,47 @@
+// The content of this file has been partially borrowed from LambdaClass Cairo VM in Rust
+// See https://github.com/lambdaclass/cairo-vm/
+
+%builtins output range_check
+from starkware.cairo.common.math import assert_le
+from starkware.cairo.common.serialize import serialize_word
+
+func signed_div_rem{range_check_ptr}(value, div, bound) -> (q: felt, r: felt) {
+    let r = [range_check_ptr];
+    let biased_q = [range_check_ptr + 1];  // == q + bound.
+    let range_check_ptr = range_check_ptr + 2;
+    %{
```

Comment:
Hey, I'm wondering, whyc are you defining this function here with the hint inside, instead of just importing from `from starkware.cairo.common.math import signed_div_rem`. Is there any particular reason?

Reply:
I see that you did that in all of the integration tests files. Shouldn't we import all the functionalities?

Reply:
I did that so the reader can see the function and what it does, this was just a personal choice. When importing, one would need to go to cairolang repo or rust cairo vm repo to find the cairo function, not ideal i guess